craft_parts/sources/git_source.py

```diff
@@ -80,13 +82,19 @@ class GitSource(SourceHandler):
     """
 
     source_model = GitSourceModel
-    command: ClassVar[str] = "git"
```

Comment:
I don't expect that anything would break with this, but it's technically an API change.

Would it be less of a change to make the method below a property instead?

Reply:
I did that initially but alas, classmethod properties are removed in Python 3.13 and some classmethods access this

Reply:
this would have been ideal :-)